contents/docs/self-host/deploy/upgrade-notes.md

```diff
@@ -132,3 +132,25 @@ If you didn’t make any customization to those, there’s nothing you need to d
     - `clickhouse.persistentVolumeClaim` -> `clickhouse.persistence.existingClaim`
 
     If you are overriding any of those, please make the corresponding changes before upgrading. Depending on your settings and setup, during this upgrade the ClickHouse pod might get recreated.
+
+### Upgrading from 10.x.x
+
+11.0.0 removes some legacy Helm annotations not needed anymore. By removing those and upgrading your installation, all future upgrades to stateless components should now happen without downtime (see [#179](https://github.com/PostHog/charts-clickhouse/pull/179) for more info).
+
+Before running the Helm upgrade command, please run the following script first (note: replace the `RELEASE_NAME` and `RELEASE_NAMESPACE` accordingly if you are using a custom release name/namespace):
+
+```
+#!/usr/bin/env sh
+
+RELEASE_NAME="posthog"
+RELEASE_NAMESPACE="posthog"
+
+for deployment in $(kubectl -n "$RELEASE_NAMESPACE" get deployments --no-headers -o custom-columns=NAME:.metadata.name | grep "posthog")
+do
+  kubectl -n "$RELEASE_NAMESPACE" label deployment "$deployment" "app.kubernetes.io/managed-by=Helm"
+  kubectl -n "$RELEASE_NAMESPACE" annotate deployment "$deployment" "meta.helm.sh/release-name=$RELEASE_NAME"
+  kubectl -n "$RELEASE_NAMESPACE" annotate deployment "$deployment" "meta.helm.sh/release-namespace=$RELEASE_NAMESPACE"
```

Comment:
Good question: that's because of a possible bug in Helm. When removing the hook it wasn't able to find anymore the resources it was previously managing.

Fresh installs don't need to follow upgrade notes. Annotations/labels will be applied automatically.